### PR TITLE
Added adp operator

### PR DESCRIPTION
**Pull request by james-cccc**
Added required manifests for the oadp operator

### CODE REVIEW

File: openshift-api-for-data-protection-operator/base/namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    kubernetes.io/metadata.name: openshift-adp
+  name: openshift-adp
+spec:
```
Comment (pittar) [inline openshift-api-for-data-protection-operator/base/namespace.yaml:7]: Please remove the "spec" stanza, it's not required.

File: openshift-api-for-data-protection-operator/base/subscription.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  labels:
```
Comment (pittar) [inline openshift-api-for-data-protection-operator/base/subscription.yaml:4]: Please remove "labels" stanza, it will be added automatically upon creation.

**Comment by pittar** [comment]
@james-cccc , looks good - just two small updates if you don't mind :)

Thanks!

File: openshift-api-for-data-protection-operator/base/namespace.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  finalizers:
```
Comment (strangiato) [inline openshift-api-for-data-protection-operator/base/namespace.yaml:8]: Finalizers are also not recommended to include.

k8s will add the necessary finalizers automatically.

Comment (james-cccc) [reply openshift-api-for-data-protection-operator/base/namespace.yaml]: Thanks, will keep that in mind :)

File: openshift-api-for-data-protection-operator/base/subscription.yaml (hunk 2 above)
Comment (james-cccc) [reply openshift-api-for-data-protection-operator/base/subscription.yaml]: Np, will keep that in mind :)

File: openshift-api-for-data-protection-operator/base/namespace.yaml (hunk 1 above)
Comment (james-cccc) [reply openshift-api-for-data-protection-operator/base/namespace.yaml]: Done

**Comment by pittar** [comment]
Thanks for the contribution!

**Pull request merged**